studiopress/genesis-sample
# Update/wp 5.4

**Added**
* CSS to adjust left padding for WordPress 5.4 Social Icons.
* Editor styles to ensure theme colors work for inline text colors.
* Editor styles for WordPress 5.4 compatibility, including link color, buttons, headings, and lists.

**Changed**
* Updated one-click theme setup content to include WordPress 5.4 button markup.

**Removed**
* Unused `light` heading class.

### How to test

1. Create a page or post and switch to the code editor.
2. Paste the block code shared below.
3. Switch back to the visual editor.
4. Follow the guidance provided below to test in the editor and the front end.

```

<h2>New Social Links Block</h2>



<p>Make sure these social icons look the way they should in your theme:</p>



<ul class="wp-block-social-links">









</ul>



<p>The social links list above should not have left padding or margin.</p>



<h2>New Buttons Block</h2>



<p>Here's a button from WordPress 5.3. Make sure it looks and is positioned the way it should in your theme:</p>



<div class="wp-block-button alignright"><a class="wp-block-button__link" href="https://google.com">5.3 button</a></div>



<p>Note: buttons where no alignment was selected would align left by default in versions prior to 5.4. They will align center in the editor now, but not on the front end. </p>



<p>Here's some buttons using the new "Buttons" block in WordPress 5.4. Make sure they are aligned properly, according to the text on each button.</p>



<div class="wp-block-buttons alignright">
<div class="wp-block-button"><a class="wp-block-button__link">5.4 Button Aligned Right</a></div>
</div>



<div class="wp-block-buttons alignleft">
<div class="wp-block-button"><a class="wp-block-button__link">5.4 button aligned left</a></div>
</div>



<div class="wp-block-buttons aligncenter">
<div class="wp-block-button"><a class="wp-block-button__link">5.4 buttons aligned center</a></div>
</div>



<div class="wp-block-buttons alignleft"></div>



<hr class="wp-block-separator clear"/>



<h2>Inline text colors</h2>



<p>Make sure the text in the following block is colored properly.</p>



<p>This text should be a normal default color. <span class="has-inline-color has-theme-primary-color">This text should be the theme's primary color.</span></p>



<p>This text should be a normal default color. <span class="has-inline-color has-theme-secondary-color">This text should be the theme's secondary accent color, if it has one.</span> If it doesn't, the text will be default.</p>



<h2>List styling changes</h2>



<p>Make sure this list looks correct in the editor and on the front end.</p>



<ul><li>Thing 1</li><li>Thing 2</li><li>Thing 3</li><li>Thing 4</li></ul>



<ol><li>Thing 1</li><li>Thing 2</li><li>Thing 3</li><li>Thing 4</li></ol>



<h2>Link color classname change</h2>



<p><a href="https://google.ca">This is a link. Does the color match what you've set for the theme's primary color in the customizer?</a></p>

```

### Documentation



Needs support documentation cc @susannelson 

### Suggested changelog entry

- See above

### Notes

None
